core-blocks/audio/edit.js

```diff
@@ -53,6 +60,14 @@ class AudioEdit extends Component {
 			this.setState( { editing: false } );
 		};
 
+		const onToggleAutoplay = ( newVal ) => {
```

Comment:
Can we use `newValue` here instead of `newVal`?

Reply:
Handled in 275ef72d3b349410d62d4c4880330cb977607932